core/src/main/java/org/bitcoinj/params/AbstractBitcoinNetParams.java

```diff
@@ -45,6 +45,7 @@
     int uahfHeight = 478559;
     /** Activation time at which the cash HF kicks in. */
     protected long cashHardForkActivationTime;
+    protected int daaHeight;
```

Comment:
What does this variable stand for ?

Reply:
That was commit merged from upstream (hash engineering repo) but https://github.com/bitcoincash-wallet/bitcoincashj/issues/2 was the issue.
The only the commit I've done is https://github.com/mycelium-com/bitcoincashj/commit/f5cca4f775c4794841c726cc49aea4a99eddc2aa

Reply:
It's hard forks block height?